packages/ffe-form-react/src/InputGroup.spec.js

```diff
@@ -174,4 +211,21 @@ describe('<InputGroup>', () => {
             'Don\'t use both "tooltip" and "description" on an <InputGroup />, pick one of them',
         );
     });
+
+    it('sets aria-describedby with two ids when both fieldMessage and description is defined', () => {
```

Comment:
Veldig pirkete med burde det være "fieldMessage and description **are** defined" i stedet for is på linje 215 og 227? :smile: